docs/resources/dli_global_variable.md

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# flexibleengine_dli_global_variable
+
+Manages a DLI global variable resource within FlexibleEngineCloud.  
```

Comment:
```suggestion
Manages a DLI global variable resource within FlexibleEngine.  
```

Reply:
from huaweicloud not huawei to FlexibleEngineCloud, keep them consistent